CHANGELOG.md

```diff
@@ -8,6 +8,20 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+## [0.3.3] - 2023-01-31
+
+[Full changelog](https://github.com/openfga/openfga/compare/v0.3.2...v0.3.3)
+
+### Added
+
+* Environment variable namings have been updated (#472). For example, `OPENFGA_MAXTUPLESPERWRITE` may now be referred to as `OPENFGA_MAX_TUPLES_PER_WRITE`. For the full list please see [.config-schema.json](./.config-schema.json). For the moment we still support the old form, but that will be dropped in a future version.
```

Comment:
```suggestion
* Environment variable names have been updated (#472).
  For example, `OPENFGA_MAX_TUPLES_PER_WRITE` instead of `OPENFGA_MAXTUPLESPERWRITE`.
  For the full list please see [.config-schema.json](https://github.com/openfga/openfga/blob/main/.config-schema.json).
  The old form still works but is considered deprecated and should not be used anymore.
```

Reply:
Thank you!